integreat_cms/cms/forms/pages/page_translation_form.py

```diff
@@ -23,3 +27,26 @@ class Meta:
         model = PageTranslation
         #: The fields of the model which should be handled by this form
         fields = MachineTranslationForm.Meta.fields + ["slug", "hix_score"]
+
+    def clean_slug(self):
+        """
+        Validate the slug field (see :ref:`overriding-modelform-clean-method`)
+
+        :return: A validated slug
+        :rtype: str
+        """
+        cleaned_slug = self.cleaned_data["slug"]
+        is_root_slug = not self.instance.ancestor_path
+        if is_root_slug and any(
+            map(
+                lambda reserved_pattern: cleaned_slug == reserved_pattern,
+                settings.RESERVED_REGION_PAGE_PATTERNS,
+            )
+        ):
+            raise ValidationError(
+                _("%(slug)s is a reserved slug."),
+                code="invalid",
+                params={"slug": cleaned_slug},
```

Comment:
```suggestion
                _("{} is a reserved slug.").format(cleaned_slug),
                code="invalid",
```
I'm aware that this goes against the recommendations of the [Django docs](https://docs.djangoproject.com/en/3.2/ref/forms/validation/#raising-validationerror), but I think `{}` as placeholder is much more readable than the oldschool `%` formatting :sweat_smile: 
Also, for consistency reasons, since this is the formatting syntax we're using everywhere else in the project...

Reply:
Linty doesn't like this -> `consider-using-f-string`

Reply:
Unfortunately, [gettext doesn't work with f-strings](https://savannah.gnu.org/bugs/?61596), so the `{}` syntax is as close as it gets.
But for me, pylint is correctly able to detect that using an f-string here is not an option... :thinking: 
What did you mean by "Linty", is this a separate tool to check the style conventions, or just a typo?

Reply:
Oops a typo. At least locally i get the warning with pylint. I've commited it nevertheless. If it works in the ci this should be fine.